pipelines/jobs/configurations/jdk19u_pipeline_config.groovy

```diff
@@ -171,7 +171,7 @@ class Config19 {
                 os                  : 'windows',
                 arch                : 'aarch64',
                 crossCompile        : 'x64',
-                additionalNodeLabels: 'win2016&&vs2019',
+                additionalNodeLabels: 'win2022&&vs2019',
```

Comment:
do we still wanna keep jdk19?

Reply:
we never shipped it anyway but let's update it for consistency @zdtsw